Unbound.cs

```diff
@@ -171,6 +170,11 @@ void Update()
             {
                 showModUi = !showModUi;
             }
+
+            if (Input.GetKeyDown(KeyCode.F2))
+            {
+                UnityEngine.SceneManagement.SceneManager.LoadScene(UnityEngine.SceneManagement.SceneManager.GetActiveScene().name);
```

Comment:
Reloading the current scene on start if holding F2... Is this leftover debug code?

Reply:
Ah, good catch, yes it is.